examples/rn-mbnb/backend/auth/providers.json

```diff
@@ -0,0 +1,7 @@
+{
+    "api-key": {
```

Comment:
I believe the should also be anonymousLogin here or the README should mention setting up login with API Key? Seems like the app is trying to login anonymously so I had to enable it manually in App Services as the the config didn't do that

Reply:
Odd that it didn't show up here.  I'll see what I can do to make that happen

Reply:
Fixed